gradle.properties

```diff
@@ -15,18 +15,15 @@ org.gradle.parallel.threads=8
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 org.gradle.parallel=true
-org.gradle.java.home=/Applications/Android Studio.app/Contents/jre/jdk/Contents/Home
 # ---------------------------------- MiniApp SDK flags ------------------------------------
 # Artifactory information needed.
 group=com.rakuten.tech.mobile.miniapp
 name=miniappplatform
-#"-" indicates aar file will be uploaded to snapshot.
-version=0.0.1-SNAPSHOT
-versionCode=1
 url=https://rakutentech.github.io/android-miniapps
 # MiniApp SDK info for Android.
 application_id=com.rakuten.mobile.miniappsplatform
 description=Mini App platform can download mini apps and display them.
+MINI_APP_BASE_URL=www.example.com
```

Comment:
Maybe the default should be defined inline in `build.gradle`? But I guess it doesn't really matter for now because we need to change this so that the base url is defined by the App which is implementing the SDK.

Reply:
Right, this will be out from here once we start doing the code architecture refactoring